### DISCUSSION TITLE
Ideas or recommendation for updating an entiity after save but not ending in a infinite loop

### DISCUSSION THREAD

**Discussion by jepzen** [koenbeuk/EntityFrameworkCore.Triggered#205] [Q&A]
I am trying to solve a problem where every time I save an entity of a specific  type i need to update a property and the save it again. But i assume that will cause another after save trigger to be triggered and that will continue to in a loop? I guess one way to be use a second db context defined without the trigger, but i rather if i could go another way. Anyone solved/thought about this kind of problem?

**Comment by jepzen** [comment]
oh maybe i should try this SaveChangesWithoutTriggers...

**Reply by koenbeuk** [reply]
`SaveChangesWithoutTriggers` could work for you. Alternatively: Scoped triggers can store properties locally (like for which entities they have already ran). Or you can store additional state in `context.Items` which is unique for each entity but which exists within a larger SaveChanges session.

Happy to clarify either of these alternative approaches
